app/controllers/format_photos_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class FormatPhotosController < ApplicationController
+  before_action :bundle_format_photo, only: %i[show edit update destroy]
+
+  def index
+    @format_photo = FormatPhoto.all
+  end
+
+  def new
+    @tformat_photo = FormatPhoto.new
```

Comment:
what is mean tformat_photo?

please listen https://www.buzzsprout.com/238868/899519- about this problem

Reply:
fixed